drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -95,6 +95,38 @@ const char *wifi_nrf_get_drv_version(void)
 	return NRF700X_DRIVER_VERSION;
 }
 
+/* If the interface is not Wi-Fi then errors are expected, hence the use of debugs */
+struct wifi_nrf_vif_ctx_zep *wifi_nrf_get_vif_ctx(struct net_if *iface)
+{
+	struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = NULL;
+	struct wifi_nrf_ctx_zep *rpu_ctx = &rpu_drv_priv_zep.rpu_ctx_zep;
+	int i;
+
+	if (!iface) {
+		LOG_DBG("%s: iface is NULL\n", __func__);
+		return NULL;
+	}
+
+	if (!rpu_ctx) {
+		LOG_DBG("%s: rpu_ctx is NULL\n", __func__);
+		return NULL;
+	}
+
+	for (int i = 0; i < ARRAY_SIZE(rpu_ctx->vif_ctx_zep); i++) {
+		if (rpu_ctx->vif_ctx_zep[i].zep_net_if_ctx == iface) {
+			vif_ctx_zep = &rpu_ctx->vif_ctx_zep[i];
+			break;
+		}
+	}
+
+	if (!vif_ctx_zep) {
+		LOG_DBG("%s: Event received for an interface that is not "
```

Comment:
For multi-iface hardware this is going to create a log message for every IPv4 event raised on any non-WiFi iface

You mention `If the interface is not Wi-Fi then errors are expected`, but this seems wrong to me -- It is very much expected that these events will be raised on non-Wi-Fi ifaces, so I'm not sure this should be a debug log

Reply:
True, but debugs are disabled by default, this is just a FYI, but if that is too much noise I can remove them.

Reply:
I'm not sure how much noise it will be, I worry mostly that it will be misleading debug output, especially given the `If the interface is not Wi-Fi then errors are expected` comment, I'd recommend at least removing that comment since it is IMO a bit misleading

If you add the IPV4 event filtering I mention in my other comment, then I think the event overlap will be reduced and this won't be so noisy so it's probably fine

Reply:
Well, after rethinking changed my mind, not much value in those prints, so, removed them now.